src/pages/gift/claim/VerifySecret.js

```diff
@@ -29,7 +29,7 @@ export default function VerifySecret ({ claimGiftHandler }) {
       <Card.Body className="d-flex flex-column">
         <CardHeader
           title={'Claim Your Gift'}
-          cardText={'Enter the <b>secret gift hash</b> you received in your email.'}
```

Comment:
Using "secret gift hash" was feedback from Michaelis, using a very distinct term for a new user to recognise. But if we use gift secret everywhere else, then I guess that's fine.